tests/returntypes/jax/test_jax_new.py

```diff
@@ -491,8 +491,8 @@ def cost(a, b):
         expected = -2 * np.sin(2 * a)
         assert np.allclose(jac, expected, atol=tol, rtol=0)
 
-    def test_grad_with_backward_mode(self, execute_kwargs):
-        """Test jax grad for adjoint diff method in backward mode"""
+    def test_grad_with_no_grad_on_execution(self, execute_kwargs):
+        """Test jax grad for adjoint diff method in no grad on execution."""
```

Comment:
Some of the `execute_kwargs` parameters for this test use `mode="forward"`. Wouldn't that mean that `grad_on_execution==True`?

Reply:
You are right I think the name of the test is very bad I will change that.